### PR TITLE
Accessibility jakaria kender

### PR DESCRIPTION
**Pull request by JakariaJishan**
For this pull request, we check :
1-Page titles  - issues found
2-Image text alternatives -issues found
3-Text headings -no issues found
4-Color contrast -no issues found
5-Resize -no issues found
6-Interaction -issues found
7-Moving content -no issues found
8-Multimedia -no issues found
9-The basic structure of the page -no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,18 +6,18 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <title>Portfolio</title>
+    <title>Portfolio Jakaria Jishan</title>
```
Comment (JakariaJishan) [inline index.html:9]: Change title from Portfolio to Portfolio Jakaria Jishan

File: index.html [hunk 2]
```diff
@@ -6,18 +6,18 @@
...
 </head>
 
 <body>
     <nav>
         <header>
             <a href="#">
-                <img class="brand-img" src="./images/Brand-logo.png" alt="brang-logo" />
+                <img class="brand-img" src="./images/Brand-logo.png" alt="brand logo" />
```
Comment (JakariaJishan) [inline index.html:16]: Change brand logo alt

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @team, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [ ] Hi @team, great job working on the accessibility aspect of the portfolio project, but I would kindly suggest you give the `paragraph` text in the `Multi-purpose Stories` of the `Most-recent work` as well as the `about-section` some increment in `font size` when you `hover` on it for easy readability and accessibility.

To look something like this 👇 

text

- [ ] I would suggest you give the `See project` and `Get my resume` some hover effect and the text some increment in `font size` for easy accessibility and readability

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi Team! @JakariaJishan @kensteph, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**